docs/docs-ref-autogen/excel/excelscript/excelscript.customconditionalformat.yml

```
@@ -4,7 +4,36 @@ uid: 'ExcelScript!ExcelScript.CustomConditionalFormat:interface'
 package: ExcelScript!
 fullName: ExcelScript.CustomConditionalFormat
 summary: Represents a custom conditional format type.
-remarks: ''
+remarks: |-
+
+
+  #### Examples
+
+  ```TypeScript
+  /**
+   * This script applies a three-color conditional formatting to the selected range.
```

Should this be "3-color" to match the Excel UI? 
```suggestion
   * This script applies a 3-color conditional formatting scale to the selected range.
```

---

This one is not applying a color scale, but rather three separate custom conditional formatting rules. Since there isn't a matching UI description relating to color, I'll leave this one a is.

---

Actually, I'll add the word "custom" to the description to better describe the UI equivalent.